### ISSUE TITLE
Delayed file download with Virtual Drive

### ISSUE THREAD

**Issue by paolodaniele** [owncloud/client#7517]
### Expected behaviour

After sync complete with virtual files i suppose to double click on that file and it opens

### Actual behaviour

After click on file it starts to check everything in the folder and files open after 10min (i've a large folder about 400GB)

### Server configuration
Operating system: Debian 10 (under KVM VM)

Web server:  Apache

Database: MariaDB

PHP version: 7.2.22

ownCloud version: latest stable (10.2.1)

Storage backend (external storage): local storage

### Client configuration
Client version: Latest Stable (2.5.4)

Operating system: Windows 10

OS language: ITA

### Logs

1. No error founds in client log files

2. Web server error log: No relevant error founds

3. Server logfile: ownCloud log (data/owncloud.log): No entry

**Comment by michaelstingl** [comment]
@paolodaniele are there ongoing transfers that need to finish before? Could you check also with the [2.6.0 RC1](https://github.com/owncloud/client/releases/tag/v2.6.0-rc1)?

**Comment by paolodaniele** [comment]
> @paolodaniele are there ongoing transfers that need to finish before? Could you check also with the [2.6.0 RC1](https://github.com/owncloud/client/releases/tag/v2.6.0-rc1)?

@michaelstingl nope. When there was ongoing transfer you could go home, have a drink, have a nap, come back to office and then maybe the file was opened :)
I haven't tried the RC but i will now and let you know.

**Comment by michaelstingl** [comment]
> When there was ongoing transfer you could go home, have a drink, have a nap, come back to office and then maybe the file was opened :)

Hm, needs some re-scheduling of transfers in the future. VFS-downloads should be more "real-time" than background syncs…

**Comment by michaelstingl** [comment]
@paolodaniele I changed he title. Do you agree?

**Comment by guruz** [comment]
The rc1 is quite outdated (we will build a rc2 soon), please stay tuned..

**Comment by ogoffart** [comment]
2.6 might help.
Also was the client running before? Was it in the synchronized state?

**Comment by paolodaniele** [comment]
> 2.6 might help.
> Also was the client running before? Was it in the synchronized state?

Nope.
First time, first clean install no updates or others. Sync has been completed. Now i'm installing it on a clean machine with 2.6.0 rc1 and it has started syncing files. I'll let you now as soon as it finishes

**Comment by guruz** [comment]
@paolodaniele Thanks for testing but the rc1 is too old.
Try this daily build from this morning: https://download.owncloud.com/desktop/daily/ownCloud-2.6.0.12865.12539-daily20191011.msi

**Comment by paolodaniele** [comment]
hi guys sorry for delayed answer. I've tried the latest build. Sync is faster 3x-5x times compared to the other version so you've done a great job on this side. But i've the same problem. I've tested with two accounts one that can see a folder (about 16GB) and in this case sync and file downlaod are very fast.
Problems start when i try with an account that can see a big folder (500GB - 400k files doc,xls,html) in this case sync with .owncloud is very faster, but file download is very slow and in some cases it cause an explorer.exe crash. If you want some log or something that maybe help let me know! Thanks!

**Comment by paolodaniele** [comment]
Ok guys, sorry for double post but i've some update.
I've run the latest release on a clean win, erased all data and log and config files.
This time sync goes at the same speed and download took about 10sec that is enought for me, but if i'm going to download a file when check is in progress it tooks 13-15min to download it. There's something that i can do for this? Because if you imagine someone who don't know this things, service is not really useful. Please let me know if you want me to do some tests.
